package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/EppoSdkRelay.kt

```diff
@@ -0,0 +1,27 @@
+package cloud.eppo.android.sdkrelay
+
+import android.app.Application
+import android.util.Log
+import cloud.eppo.android.EppoClient
+
+class EppoSdkRelay : Application() {
+
+  val API_KEY: String = BuildConfig.API_KEY // Set in root-level local.properties
+
+  override fun onCreate() {
+    super.onCreate()
+    EppoClient.Builder(API_KEY, this)
```

Comment:
~Are we initializing the client twice? Once here and once in the Test Client Activity? Are both needed?~

Nevermind its for re-initializing.

I wonder if we don't do this here and just have a shared method we call when force-reintializing that we also call at the start of that activity. Would reduce duplication and better spatially locate the two places we (re)initialize the SDK

Reply:
yep. not needed here at all